src/commands/vault/show.ts

```diff
@@ -0,0 +1,46 @@
+import { Kommand } from '../../common'
+import { flags } from '@oclif/command'
+import * as _ from 'lodash'
+import chalk from 'chalk'
+
+// tslint:disable-next-line
```

Comment:
Same proposal

When disabling a eslint rule may it be better to specify which one you disabled ?

Like // tslint:disable-next-line no-...